core/arch/x86/clean_call_opt.c

```diff
@@ -74,7 +74,10 @@ analyze_callee_regs_usage(dcontext_t *dcontext, callee_info_t *ci)
          * impact unless there are a lot of different clean call callees.
          */
         /* XMM registers usage */
-        for (i = 0; i < MCXT_NUM_SIMD_SLOTS; i++) {
+        /* XXX: This looks like it is missing support for AVX/AVX2.
```

Comment:
Why do you say that?  Maybe the comment should be more specific about what is missing.

Reply:
I thought it is missing ymms but I realize that instr_uses_reg(instr, DR_REG_XMM0 + i) is true if the simd reg is ymm (or future zmm). I've removed both comments, this should work as-is.